webapp-django/crashstats/crashstats/models.py

```diff
@@ -384,7 +390,10 @@ def fetch(
         implementation_method = getattr(implementation, method)
         result = implementation_method(**params)
         if cache_key:
-            cache.set(cache_key, result, self.cache_seconds)
+            try:
+                cache.set(cache_key, result, self.cache_seconds)
+            except MemcacheServerError:
+                metrics.incr("cache_set_error")
```

Comment:
We probably want to know what other crash ids this happens with. I'll figure out how to log this later.